agent_guard_core/cli.md

```diff
@@ -0,0 +1,61 @@
+# Agent Guard CLI
+
+The Agent Guard CLI provides commands to configure and manage secret providers and related options for Agent Guard.
+
+## Usage
+
+```sh
+agent-guard [COMMAND] [OPTIONS]
+```
+
+## Commands
+
+### configure
+
+Group of commands to manage Agent Guard configuration.
+
+#### set
+
+Set the secret provider and related Conjur options.
+
+**Options:**
+- `--provider [PROVIDER]`  
+  The secret provider to store and retrieve secrets.  
+  Choices: `AWS_SECRETS_MANAGER_PROVIDER`, `FILE_SECRET_PROVIDER`, `CONJUR_SECRET_PROVIDER`  
+  Default: `FILE_SECRET_PROVIDER`
+
+- `--conjur_authn_login [LOGIN]`  
+  (Optional) Conjur authentication login (workload ID).
+
+- `--conjur_authn_api_key [API_KEY]`  
+  (Optional) API Key to authenticate to Conjur Cloud.
+
+- `--conjur_appliance_url [URL]`  
+  (Optional) Endpoint URL of Conjur Cloud.
+
+**Example:**
+```sh
+agent-guard configure set --provider CONJUR_SECRET_PROVIDER --conjur_authn_login my-app --conjur_authn_api_key my-key --conjur_appliance_url https://conjur.example.com
```

Comment:
The word "configure" is a verb, but since this is a category for other commands, maybe it should be a noun? I think we can do either "configuration" in long or "config" in short. I prefer "config"